quickstarts/elastic-vector-search/config.yml

```diff
@@ -0,0 +1,56 @@
+
+slug: elastic-vector-search
+description: |
+  ## Why should you monitor your usage of Elastic Vector Search?
+
+  Monitor your vector searches on Elastic Vector Search to get visibility on what you send to Elastic Vector Search, responses retrieved from Elastic Vector Search, latency, usage and errors. 
+
+  ### Track the query performance of your Vector DB
+  Track the behavior of your vector stores. Monitor the latency, queries, the number of documents retrieved, and the content of the documents so that you can evaluate their relevance.
+ 
+  ### Track your app:  
+  By tracking key metrics like latency, throughput, error rates, and input & output, you can gain insights into your app's performance and identify areas of improvement.
+
+
+  ### What’s included in this quickstart?
+
+  New Relic LangChain monitoring quickstart provides a variety of pre-built dashboards, which will help you gain insights into the health and performance of your LangChain app. These reports include:
```

Comment:
```suggestion
  Elastic Vector Search uses New Relic Langchain monitoring which provides a variety of pre-built dashboards, which will help you gain insights into the health and performance of your Elastic Vector Search app. These reports include:
```

Reply:
Hi @rahulbasu-nr ,
Updated the review changes. Kindly review it.

Thanks...